### ISSUE TITLE
[Python] Bug with required properties that are nullable

### ISSUE THREAD

**Issue by elias4335** [swagger-api/swagger-codegen#12021]
##### Description
When I use a method that returns properties that can be "nullable" the response classes setter method raises an error like this: "raise ValueError("Invalid value for `vin`, must not be `None`")". But since the property `vin` is set to nullable = true in the specification there should be no errror.

##### Swagger-codegen version

Version 3.0.36

Open API spec
  --> https://gist.github.com/elias4335/94651f2efd49111b4344a5109da8b515

##### trouble code
```
 @vin.setter
    def vin(self, vin):
        """Sets the vin of this GetMachinesResponse.

        Machine Vehicle Identification Number  # noqa: E501

        :param vin: The vin of this GetMachinesResponse.  # noqa: E501
        :type: str
        """
        #if vin is None:
        #    raise ValueError("Invalid value for `vin`, must not be `None`")  # noqa: E501

        self._vin = vin
```

For now I just turned the code into comments.

**Comment by MalteEbner** [comment]
This is probably going to be solved by https://github.com/swagger-api/swagger-codegen/pull/10906

**Comment by elias4335** [comment]
Thanks

**Comment by mohit1337** [comment]
Issue is marked as closed but mentioned https://github.com/swagger-api/swagger-codegen/pull/10906 was never merged

**Comment by thibgc** [comment]
Good news! #10906 was revived as #12630, merged into master, and now part of release 2.4.49 two days ago! Thank you @daniel-kmiecik! 🥳
